### DISCUSSION TITLE
umi3.5项目过大时热更新速度过慢(进度条会卡在copy-webpack-plugin 92% 这里)，有无解决办法？

### DISCUSSION THREAD

**Discussion by IsDyh01** [umijs/umi#12691] [Q&A]
![image](https://github.com/user-attachments/assets/02922d9e-2f9f-4ecf-9286-92e9d6b6a97c)
![image](https://github.com/user-attachments/assets/36008f3b-517c-4368-8b7a-27fccb43ff22)

**Comment by xierenyuan** [comment]
试试 mako https://github.com/umijs/umi-plugin-mako
